com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsLighting.hlsl

```diff
@@ -1,6 +1,6 @@
 // Copyright � 2020 Unity Technologies ApS
 // Licensed under the Unity Companion License for Unity-dependent projects--see https://unity3d.com/legal/licenses/Unity_Companion_License
-// Unless expressly provided otherwise, the Software under this license is made available strictly on an �AS IS� BASIS WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED. Please review the license for details on these and other terms and conditions.
+// Unless expressly provided otherwise, the Software under this license is made available strictly on an �AS IS�EBASIS WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED. Please review the license for details on these and other terms and conditions.

```

Comment:
Nitpick, looks like an 'E' was added.

```suggestion
// Unless expressly provided otherwise, the Software under this license is made available strictly on an AS IS BASIS WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED. Please review the license for details on these and other terms and conditions.

```